eZ/Publish/Core/Persistence/Elasticsearch/Content/Search/Mapper.php

```diff
@@ -25,101 +25,19 @@
 /**
  * Mapper maps Content and Location objects to a Document object, representing a
  * document in Elasticsearch index storage.
+ *
+ * Note that custom implementations might need to be accompanied by custom mappings.
  */
-class Mapper
```

Comment:
Why did you go for an abstract and not an interface, given that it has no concretion whatsoever ?